### PR TITLE
KSP-64k Netkan Initial Commit

### PR DESCRIPTION
**Pull request by Felger**

### CODE REVIEW

**Comment by Felger** [comment]
Accidentally modified my master when updating the NearFuture pack stuff. Undid the changes, should be untouched at this point.

File: NetKAN/KSP-64k-imgviewer.netkan [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "install" : [
+        {
+            "file"       : "GameData/img_viewer",
+            "install_to" : "GameData"
+        }
+    ],
+    "depends" : [
+        { "name" : "ModuleManager"},
+        { "name" : "KSP-64k"},
+	{ "name" : "Kopernicus"}
```
Comment (dbent) [inline NetKAN/KSP-64k-imgviewer.netkan]: Missing comma

**Comment by Felger** [comment]
Updated identifiers, removed unnecessary dependencies.

**Comment by Dazpoet** [comment]
Seems some small part of changes to another file slipped in so I can't really work with this since my gitomancy level is too low.

**Comment by RichardLake** [comment]
@Felger There is still white-space changes to NetKAN/NearFutureElectrical.frozen - could you remove them?

**Comment by dbent** [comment]
Closing in favor of #1754, #1755, and #1756. Basically the same thing except using KS as the source.

**Pull request closed without merging**